From 77d5f6baf8b02e9ac118880c59cadfcad32f7ee5 Mon Sep 17 00:00:00 2001
From: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Date: Thu, 28 Feb 2019 05:46:29 -0800
Subject: [PATCH 0533/1851] dt-bindings: ASoC: xlnx,spdif: update input clocks

commit a7ca69066f11b1e6a70b6ef53a5a7b0de0d84422 from
https://github.com/Xilinx/linux-xlnx.git

Update input clocks bindings of SPDIF Tx and SPDIF Rx.

Signed-off-by: Maruthi Srinivas Bayyavarapu <maruthi.srinivas.bayyavarapu@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 .../devicetree/bindings/sound/xlnx,spdif.txt       | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/Documentation/devicetree/bindings/sound/xlnx,spdif.txt b/Documentation/devicetree/bindings/sound/xlnx,spdif.txt
index 600ab5cbefcd..45214da42ec7 100644
--- a/Documentation/devicetree/bindings/sound/xlnx,spdif.txt
+++ b/Documentation/devicetree/bindings/sound/xlnx,spdif.txt
@@ -4,20 +4,20 @@ The IP supports SPDIF based playback and capture audio
 
 Required properties:
  - compatible: "xlnx,spdif-2.0"
- - clocks: List of phandles to AXI4-Lite clock,
-	   Audio clock input used at the SPDIF/AES3 interface
- - clock-names: Must contain "s_axi_aclk", "aud_clk_i" in the same order as
-		clocks listed in clocks property
+ - clock-names: List of input clocks.
+   Required elements for SPDIF Tx: "aud_clk_i", "s_axi_aclk", "s_axis_aclk".
+   Required elements for SPDIF Rx: "aud_clk_i", "s_axi_aclk", "m_axis_aclk".
+ - clocks: Input clock specifier. Refer to common clock bindings.
  - reg: Base address and address length of the IP core instance.
  - interrupts-parent: Phandle for interrupt controller.
  - interrupts: List of Interrupt numbers.
  - xlnx,spdif-mode: 0 :- receiver mode ; 1 :- transmitter mode
  - xlnx,snd-pcm: phandle to audio formatter node
 
-Example:
+Example - SPDIF Rx:
 	spdif_0: spdif@80010000 {
-		clock-names = "aud_clk_i", "s_axi_aclk";
-		clocks = <&si570_1>, <&clk 71>;
+		clock-names = "aud_clk_i", "s_axi_aclk", "m_axis_aclk";
+		clocks = <&si570_1>, <&clk 71>, <&clk 71>;
 		compatible = "xlnx,spdif-2.0";
 		interrupt-parent = <&gic>;
 		interrupts = <0 91 4>;
-- 
2.31.1

